amp-pwa-reader/src/js/Article.js

```diff
@@ -155,17 +156,19 @@ class Article {
 
     // No animation if there's no card to animate from
     if (!this.card) {
-        return new Promise(resolve => resolve());
+      return new Promise(resolve => resolve());
     }
 
-    let offset = (innerWidth * this.card.imageData.ratio) / 2;
-    this.container.style.transform = 'translateY(' + (offset + scrollY) + 'px)';
-
     return new Promise(resolve => {
-      return this.container.animate([
-        { opacity: 1, transform: 'translateY(' + (scrollY) + 'px)' },
-        { opacity: 0, transform: 'translateY(' + (offset + scrollY) + 'px)' }
-      ], { duration: this.cssVariables.animationSpeedOut, easing: 'cubic-bezier(0.23, 1, 0.32, 1)' }).onfinish = resolve;
+
+      this._transitionEndCallback = () => {
```

Comment:
If you are creating and destroying this function in this one scope, is there a reason to save it as a `this` var?